qa/tasks/ceph.py

```diff
@@ -1585,6 +1585,14 @@ def restart(ctx, config):
     if config.get('wait-for-osds-up', False):
         for cluster in clusters:
             ctx.managers[cluster].wait_for_all_osds_up()
+            if (expected_fail = config.get('expected-failure') is not None):
+                stdout = ctx.managers[cluster].sh(
+		    args = [
+                        'sudo',
+                        'egrep', pattern,
+                        '/var/log/ceph/{cluster}.log'.format(cluster=cluster),
```

Comment:
I doubt we'll have an access the monitor at time when OSD calls `ObjectStore::create()` – `MonClient` is instantiated later.

Reply:
Changing to egrep from osds logs